code/game/machinery/computer/ai_core.dm

```diff
@@ -219,7 +219,7 @@ If adding stuff to this, don't forget that an AI need to cancel_camera() wheneve
 That prevents a few funky behaviors.
 */
 //What operation to perform based on target, what ineraction to perform based on object used, target itself, user. The object used is src and calls this proc.
-/obj/item/proc/transfer_ai(var/choice as text, var/interaction as text, var/target, var/mob/U as mob)
+atom/proc/transfer_ai(var/choice as text, var/interaction as text, var/target, var/mob/U as mob)

```

Comment:
for the love of all that is holy, convert this proc to be OOP
(that means no more "MECHA" junk, just ```/obj/mecha/transfer_ai()``` etc.)